pkg/app/api/pipedstatstore/store.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2021 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipedstatstore
```

Comment:
How about passing the cache directly to the `api` instance instead of creating this store package?
Here are some reasons:
- This store is containing only cache
- Ops also needs this, but this is putting inside `api` package

Reply:
You're right 👍 lets me address this 🙏

Reply:
Addressed on https://github.com/pipe-cd/pipe/pull/2190/commits/ecfbd0d10bf607abf736b533b94996da69897c29 🙏